format.sh

```diff
@@ -9,7 +9,7 @@ shopt -s globstar
 if (( $# > 0 )); then
     echo "Formatting file(s) $*"
     echo "Running clang-format..."
-    clang-format ${FORMAT_OPTS} "$@"
+    clang-format-11 ${FORMAT_OPTS} "$@"
```

Comment:
you probably didn't mean to commit this

think of plebs like me using clang-format 6 Pepega

but it seems to have formatted some ternaries very nicely 👀 I wish it didnt go away next legacy-clang-format run

Reply:
Yeah this was an accident.  but I think we should just pick on a version to use so we dont fight each other with formatting changes.